code/datums/components/food/edible.dm

```diff
@@ -465,6 +499,24 @@ Behavior that's still missing from this component that original food items had t
 		return
 	return TRUE
 
+///Applies food buffs according to the crafting complexity
+/datum/component/edible/proc/ApplyBuff(mob/eater)
```

Comment:
Can you snake case this pls?

I know it's trying to be consistent but they're also wrong